app/src/main/res/layout/item_wallet.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Idealnaya rabota LLC
+  ~ Licensed under Multy.io license.
+  ~ See LICENSE for details
+  -->
+
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/list_item_height"
+    android:layout_marginBottom="@dimen/wallet_item_margin_height"
+    android:layout_marginEnd="@dimen/activity_horizontal_margin"
+    android:layout_marginStart="@dimen/activity_horizontal_margin"
+    android:layout_marginTop="@dimen/wallet_item_margin_height"
+    android:background="@drawable/shape_squircle_white"
+    android:elevation="@dimen/elevation_4dp">
+
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="@dimen/list_item_image"
+        android:layout_height="@dimen/list_item_image"
+        android:layout_marginStart="@dimen/margin_13dp"
+        android:src="@drawable/ic_btc"
```

Comment:
srcCompat?

Reply:
@Anschutz1927 Actually, it's not necessary to use srcCompat since we use Lollipop as minimum supported device. 

Generally srcCompat is needed to display vector drawables on pre-Lollipop devices